admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -2,11 +2,17 @@
 Config.php Parameters
 =====================
 ownCloud uses the ``config/config.php`` file to control server operations.
-``config/config.sample.php`` lists all the configurable parameters within
-ownCloud. This document provides a more detailed reference. Many options are
+``config/config.sample.php`` lists all the configurable parameters and
+their defaults within ownCloud.
+
+This document provides a more detailed reference. Many options are
 configurable on your Admin page, so it is usually not necessary to edit
 ``config/config.php``.
 
+.. note:: Both, the ``config/config.sample.php`` and this document are
+   showing the defaults of each parameter. Until you don't want to
+   change those defaults don't add it to your ``config/config.php``.
+
```

Comment:
Suggestion:
If you don't want to change the default then don't add the parameter to your ...